packages/carbon-web-components/src/components/loading/loading.ts

```diff
@@ -33,20 +33,27 @@ class BXLoading extends LitElement {
   @property()
   type = LOADING_TYPE.REGULAR;
 
+  /**
+   * `true` if overlay should be applied.
+   */
+  @property({ attribute: 'overlay' })
```

Comment:
```suggestion
    @property({ type: Boolean, reflect: true })
```

so then when it shows in the DOM either the attibute appears like `<cds-loading overlay />` and if you don't want the overlay attribute then it would just be `<cds-loading />`  instead of having it be `overlay="true"` or `overlay="false"`

this will then effect how the styles are updated

Reply:
updated the property name to match the attribute since they're now being reflected!